src/sentry/api/event_search.py

```diff
@@ -1565,6 +1571,9 @@ def add_default_arguments(self, field, columns, params):
         return columns
 
     def format_as_arguments(self, field, columns, params):
+        if self.private and self.name not in params["access"]:
+            raise InvalidSearchQuery(u"{}: no access to function".format(field))
+
```

Comment:
Should we have this error the same way as if the function didn't exist? eg. using `blah` in discover gets us an error like:
`blah() is not a valid function`